kivy/uix/scrollview.py

```diff
@@ -1148,7 +1158,7 @@ def update_from_scroll(self, *largs):
         self._bar_color = self.bar_color
         ev()
 
-    def _bind_inactive_bar_color(self, *l):
+    def _bind_inactive_bar_color(self, *args):
```

Comment:
For readability I prefer having `*_` over `*args` when args isn't used.

Reply:
Same as above this is following existing usage, in this file. While I am ok with using *_  it would stand out from other places of similar usage in the file.

Not in the scope of this change.